### ISSUE TITLE
Gaz sélectionné par défaut mais non-coché

### ISSUE THREAD

**Issue by florianpanchout** [datagir/nosgestesclimat-site#574]
Sur la page https://nosgestesclimat.fr/simulateur/bilan?cat%C3%A9gorie=logement, "Gaz" est sélectionné par défaut mais non-coché. 

Je me souviens qu'il avait été décidé que tout était décoché par défaut dans ces mosaïques. Mais ça ne fonctionne pas si les cases sont cochées de façon invisible. Surtout que si je sélectionne "Fioul Domestique", ça n'enlève pas l'impact du gaz mais cumule les deux (alors que je n'ai jamais coché le gaz)

**Comment by laem** [comment]
Oui, on met comme valeur par défaut dans la simulation que l'utilisateur a du gaz. Le gaz disparait du calcul s'il n'a pas été coché après validation de la question. 

Auparavant, les valeurs par défaut dans la mosaïque étaient présélectionnées en toute transparence, mais ça ne convenait pas à certains utilisateurs. 

L'alternative serait de garder le gaz comme valeur par défaut, puis quand on arrive sur la mosaïque chauffage, de décocher la case dans l'interface ET de mettre la valeur par défaut à zéro. 

Ça ferait du coup varier la valeur de simulation en amont de la question, plutôt qu'en aval. C'est peut-être mieux niveau expérience utilisateur, mais j'en suis pas convaincu. 

L'autre solution, qui me semble la meilleure, serait de cocher la case gaz mais avec un style dédié au "par défaut", mais le risque est que l'utilisateur ne pense pas à le décocher, et donc qu'on le lui compte.

**Comment by florianpanchout** [comment]
> Le gaz disparait du calcul s'il n'a pas été coché après validation de la question.

My bad, je n'avais pas vu cette étape. 

> L'autre solution, qui me semble la meilleure, serait de cocher la case gaz mais avec un style dédié au "par défaut", mais le risque est que l'utilisateur ne pense pas à le décocher, et donc qu'on le lui compte.

Et le mettre en grisé (comme le placeholder des questions textes), avec le bouton "Je ne sais pas" qui s'affiche (et une option "Pas de chauffage") ?

**Comment by laem** [comment]
En tout cas, la PR  #1433 touche un peu à ça je crois. 

Pour le "pas de chauffage", on peut le résoudre via les suggestions implémentées par @Clemog dans la PR alimentation en cours.

**Comment by Clemog** [comment]
On pourrait effectivement rajouter cette suggestion !
Je me pose quand même la question de la validation des mosaïques:

on a bien 2 types de mosaïques :
- les mosaïques "numériques" comme celle des repas
- les mosaïques "de sélection" comme celle des chauffages ou des équipements numériques par exemple 

La PR #188 a permis de "valider tous les choix" au clic sur "suivant" pour les mosaïques : le bouton "je ne sais pas" a donc disparu. Cette fonctionnalité fonctionne bien pour les mosaïques numériques car on voit via la style les valeurs par défaut. Néanmoins cette fonctionnalité est discutable pour les mosaïques de sélection car on ne voit plus les valeurs par défaut depuis #378 il me semble. Une autre problème : pour un utilisateur qui ne coche rien mais clique sur "suivant", n'est considéré aucun mode de chauffage (tout à 0). 

2 possibilités :

- Afficher le bouton je ne sais pas pour les mosaïques de sélection (sans les valeurs par défaut pré-cochées qui portaient à confusion)
- Laisser seulement le bouton suivant mais à la validation, ne pas mettre les sélection à 0 mais par défaut
- Afficher les valeurs par défaut en transparence + "je ne sais pas"
